.ci/.jenkins_scope.yaml

```diff
@@ -1,6 +0,0 @@
-SCOPE:
-  - "@elastic/apm-rum-core"
-  - "@elastic/apm-rum"
-  - "@elastic/apm-rum-react"
-  - "@elastic/apm-rum-angular"
-  - "@elastic/apm-rum-vue"
```

Comment:
Are `SCOPE` and `STACK_VERSION` now handled somewhere else?

Reply:
Yeah, it' defined at Jenkinsfile level, in a matrix: https://github.com/elastic/apm-agent-rum-js/blob/master/Jenkinsfile#L102-L123